templates/lune/partials/product/overlay.snip.html

```diff
@@ -0,0 +1,208 @@
+              </amp-img>
+            {{/mobile}}
+
+            {{#desktop}}
+              <!-- Desktop -->
+              <!-- WebP -->
+              <amp-img
+                alt="{{title}}"
+                srcset="{{webp_srcset}}"
+                layout="responsive"
+                width="{{width}}"
+                height="{{height}}"
+                media="(min-width: 720px)">
+                <!-- Fallback when WebP is not supported -->
+                <amp-img
+                  fallback
+                  alt="{{title}}"
+                  srcset="{{fallback_srcset}}"
+                  layout="responsive"
+                  width="{{width}}"
+                  height="{{height}}"></amp-img>
+              </amp-img>
+            {{/desktop}}
+          </figure>
+        {{/image}}
+
+        <div class="relative z1 center">
+          <h1 class="lune-overlay-hero-title h3 md-h0 mb4 mx-auto col-10 caps lune-anim-move-up">{{title}}</h1>
+
+          <!-- Meet the makers CTA -->
+          {{> ../components/cta.snip.html}}
+        </div>
+
+        <!-- Scroll indicator -->
+        {{> ../components/scroll-indicator.snip.html}}
+      </header>
+    {{/hero}}
+
+    {{#top-section}}
+      <section class="lune-overlay-content relative col-12 mx-auto flex flex-column p2 md-p0">
+        <div class="col-1 flex-none xs-hide sm-hide"></div>
+        {{#image}}
+          <figure class="relative col-12 md-col-5 flex-none m0">
+            {{#mobile}}
+              <!-- Mobile -->
+              <!-- WebP -->
+              <amp-img
+                alt="{{alt}}"
+                srcset="{{webp_srcset}}"
+                layout="responsive"
+                width="{{width}}"
+                height="{{height}}"
+                media="(max-width: 719px)">
+                <!-- Fallback when WebP is not supported -->
+                <amp-img
+                  fallback
+                  alt="{{alt}}"
+                  srcset="{{fallback_srcset}}"
+                  layout="responsive"
+                  width="{{width}}"
+                  height="{{height}}"></amp-img>
+              </amp-img>
+            {{/mobile}}
+
+            {{#desktop}}
+              <!-- Desktop -->
+              <!-- WebP -->
+              <amp-img
+                alt="{{alt}}"
+                srcset="{{webp_srcset}}"
+                layout="responsive"
+                width="{{width}}"
+                height="{{height}}"
+                media="(min-width: 720px)">
+                <!-- Fallback when WebP is not supported -->
+                <amp-img
+                  fallback
+                  alt="{{alt}}"
+                  srcset="{{fallback_srcset}}"
+                  layout="responsive"
+                  width="{{width}}"
+                  height="{{height}}"></amp-img>
+              </amp-img>
+            {{/desktop}}
+          </figure>
+        {{/image}}
+        <div class="col-1 flex-none xs-hide sm-hide"></div>
```

Comment:
To mimimise the amount of custom css, we are using divs here with bass css columns to create gutters. The alternative is to use custom css to create margins with the same column % widths.